morphir/runtime/src/org/finos/morphir/runtime/quick/NativeFunction.scala

```diff
@@ -0,0 +1,137 @@
+package org.finos.morphir.runtime.quick
+
+import org.finos.morphir.Hints
+import org.finos.morphir.naming.*
+import org.finos.morphir.runtime.IllegalValue
+import org.finos.morphir.runtime.quick.RTValue.Primitive
+
+case class NativeContext(evaluator: Loop, hints: Hints = Hints.empty)
+
+sealed trait DynamicNativeFunction {
+  def name: String
+}
+
+// format: off
+class DynamicNativeFunction1[T1 <: RTValue, R <: RTValue](val name: String)(
```

Comment:
Don't think type bound is needed on the type especially since we have it on the apply. In general the type bounds should be on methods and not the generic types.